sdk/keyvault/azure-keyvault-keys/tests/test_crypto_client.py

```diff
@@ -237,6 +244,7 @@ def test_wrap_and_unwrap(self, **kwargs):
         self.assertEqual(key_bytes, result.key)
 
     def test_symmetric_encrypt_and_decrypt_mhsm(self, **kwargs):
+        """Encrypt and decrypt with the service"""
         is_hsm = True
         self._skip_if_not_configured(is_hsm)
         endpoint_url = self.managed_hsm_url if is_hsm else self.vault_url
```

Comment:
Looks like I missed this in an earlier PR 😿 
```suggestion
        self._skip_if_not_configured(True)
        key_client = self.create_key_client(self.managed_hsm_url)
```

Reply:
I also just noticed this while making these changes and forgot to fix it -- thanks for catching it!